src/snapshot/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "snapshot"
+version = "0.0.1"
+authors = ["Amazon Firecracker team <firecracker-devel@amazon.com>"]
+
+[dependencies]
+serde = ">=1.0.27"
+serde_derive = ">=1.0.27"
+bincode = "1.2.1"
+utils = { path="../utils" }
```

Comment:
../utils brings other dependencies like net_get, can we make sure this is really decoupled from other local crates ?

Reply:
That probably isn't needed, I have to double check.